.prow.yaml

```
@@ -78,3 +78,405 @@ presubmits:
         command:
         - "/usr/local/bin/lichen.sh"
 
+postsubmits:
+
+  
+  - name: build-push-nephio-operator-dev-image
+    cluster: default
+    always_run: true
+    branches:
+    - "main"
+    annotations:
+      description: Build and Push Nephio Operator Dev Image to DockerHub
+    decorate: true
+    decoration_config:
+      censor_secrets: true
+    max_concurrency: 1
+    extra_refs:
+    - org: nephio-project
+      repo: nephio
+      base_ref: main
+      path_alias: "nephio_build"
+    spec:
+      containers:
+      - name: kaniko
+        image: gcr.io/kaniko-project/executor:debug
+        command:
+        - "/bin/sh"
+        - "-c"
+        - |
+          executor --context=/home/prow/go/src/nephio_build/ --dockerfile=operators/nephio-controller-manager/Dockerfile --destination=nephio/nephio-operator:${PULL_BASE_REF} --destination=nephio/nephio-operator:latest
```

It would be better to use BUILD_ID here instead of PULL_BASE_REF which is not unique (Ref name of the base branch here it would be 'main') https://docs.prow.k8s.io/docs/jobs/#job-environment-variables
```suggestion
          executor --context=/home/prow/go/src/nephio_build/ --dockerfile=operators/nephio-controller-manager/Dockerfile --destination=nephio/nephio-operator:${BUILD_ID} --destination=nephio/nephio-operator:latest
```